oak/server/logging_node.cc

```diff
@@ -47,8 +48,16 @@ void LoggingNode::Run(Handle handle) {
       if (result.msg == nullptr) {
         break;
       }
-      LOG(INFO) << "{" << name_ << "} "
```

Comment:
Ah, there are actually two runtime implementations of the logging pseudo-Node &ndash; you also need to update `oak/server/rust/oak_runtime/src/node/logger.rs` to cope with proto-encoded logging messages.

Reply:
Oh! Thanks for letting me know. I'll fix it :)